### PR TITLE
Hometask1 (task1-3)

### PR DESCRIPTION
**Pull request by umar0v**

### CODE REVIEW

File: hometask1/js/classes.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        [this.firstName, this.lastName] = newName.split(' ');
+    }
+
+    /**
+     * Display Human properties
+     * @returns {string}
+     */
+    displayInfo()
+    {
+        let info = `First Name: ${this.firstName}<br>`;
```
Comment (ubcent) [inline hometask1/js/classes.js]: Why do you need concatenation here? Template strings can include line breaks without requiring use of the linefeed character. For example:
```javascript
let info = `
  First Name: ${this.firstName}<br>
  Last Name: ${this.lastName}<br>
  Age: ${this.age}<br>
  Date of Birth: ${this.dateOfBirth}<br>
`;
```

File: hometask1/js/classes.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+{
+    /**
+     * @param {string} firstName
+     * @param {string} lastName
+     * @param {int} age
+     * @param {string} dateOfBirth
+     * @param {string} department
+     * @param {int} salary
+     */
+    constructor(firstName, lastName, age, dateOfBirth, department, salary)
```
Comment (ubcent) [inline hometask1/js/classes.js]: I'd recommend using the object instead of multiple of arguments.

File: hometask1/js/classes.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    {
+        super(firstName, lastName, age, dateOfBirth);
+        this.department = department;
+        this.salary = salary;
+    }
+
+    displayInfo()
+    {
+        let info = super.displayInfo();
```
Comment (ubcent) [inline hometask1/js/classes.js]: You can call it through **this** because you called `super()` previously.

File: hometask1/js/functions.js [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+ * Calculate area of selected figure
+ *
+ * @param {Node} form - Form node where user input parameters saved
+ * @param {Node} container - Container where calculations results will be shown
+ */
+function calculateArea(form, container)
+{
+    container.innerHTML = '';
+
+    let figures = {
```
Comment (ubcent) [inline hometask1/js/functions.js]: As it seems to me this object looks excessively

File: hometask1/js/classes.js (hunk 1 above)
Comment (umar0v) [reply hometask1/js/classes.js]: Oh, that's grate. Thank you! I'll be using such method in a future.

**Pull request merged**